hooks/environment

```diff
@@ -1,2 +1,35 @@
-#!/bin/bash
-python "${BASH_SOURCE%/*}/../python/setup-hooks.py"
+#!/usr/bin/env bash
+
+declare -a hooks=(
+    # "environment" # not allowed
+    "pre-checkout"
+    "checkout"
+    "post-checkout"
+    "pre-command"
+    "command"
+    "post-command"
+    "pre-artifact"
+    "post-artifact"
+    "pre-exit"
+)
+declare -a extensions=(
+    ""
+    ".bat"
+    ".cmd"
+)
+echo "whereami $(pwd)"
+pwd
+hooks_dir="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+echo "hooks_dir: ${hooks_dir}"
+env | sort | grep METAHOOK
```

Comment:
```suggestion
```
Also seems like a debug line - its not the users business that we look this stuff up from env vars
(also its in buildkite env tab already)